test/core/physicalModel/wire/MultiWireTest.cpp

```diff
@@ -0,0 +1,128 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
```

Comment:
why this? Is it used?

Reply:
EXPECT_THAT and ASSERT_THAT are two test assertions specifically from gmock, that add a lot of new ways to check data, for example, check if a container has exactly specific elements inside.

Reply:
Ok. I didn't know about those. I have always managed without but maybe I should take a look on them.